apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -270,6 +327,12 @@ export default Vue.extend({
       actualTableHeight: "100%",
       loading: false,
 
+      // modal
+      modalName: "viewJsonModel",
```

Comment:
There is a common issue with modals in our app, if we have multiple tabs open this could display the wrong content, save to the wrong tab, etc. Every data tab will have the same modal name and they can interfere with eachother. You can check TabQueryEditor.vue for the save modal and the parameter modal for how this was fixed. It's ugly but it works :joy:

Reply:
Yes Day is right, you should name this something like `view-json-modal-${tab.id}`